core/queryalgebra/evaluation/src/test/java/org/eclipse/rdf4j/query/algebra/evaluation/impl/QueryCostEstimatesTest.java

```diff
@@ -36,31 +37,30 @@ public void testBindingSetAssignmentOptimization() throws RDF4JException {
 		QueryJoinOptimizer opt = new QueryJoinOptimizer(new EvaluationStatistics(), new EmptyTripleSource());
 		opt.optimize(q.getTupleExpr(), null, null);
 
-		assertEquals("QueryRoot" + LINE_SEP +
-				"   Projection" + LINE_SEP +
-				"      ProjectionElemList" + LINE_SEP +
-				"         ProjectionElem \"s\"" + LINE_SEP +
-				"         ProjectionElem \"p\"" + LINE_SEP +
-				"         ProjectionElem \"o\"" + LINE_SEP +
-				"         ProjectionElem \"x\"" + LINE_SEP +
-				"      Join" + LINE_SEP +
-				"         StatementPattern (costEstimate=1, resultSizeEstimate=1)" + LINE_SEP +
-				"            Var (name=_const_5c6ba46_uri, value=ex:s2, anonymous)" + LINE_SEP +
-				"            Var (name=_const_af00e088_uri, value=ex:pred, anonymous)" + LINE_SEP +
-				"            Var (name=_const_17c09_lit_e2eec718, value=\"bah\", anonymous)" + LINE_SEP +
-				"         Join" + LINE_SEP +
-				"            StatementPattern (costEstimate=10, resultSizeEstimate=10)" + LINE_SEP +
-				"               Var (name=_const_5c6ba45_uri, value=ex:s1, anonymous)" + LINE_SEP +
-				"               Var (name=_const_af00e088_uri, value=ex:pred, anonymous)" + LINE_SEP +
-				"               Var (name=v)" + LINE_SEP +
-				"            LeftJoin (new scope) (costEstimate=1000, resultSizeEstimate=1000)" + LINE_SEP +
-				"               StatementPattern (resultSizeEstimate=1000)" + LINE_SEP +
-				"                  Var (name=s)" + LINE_SEP +
-				"                  Var (name=p)" + LINE_SEP +
-				"                  Var (name=o)" + LINE_SEP +
-				"               BindingSetAssignment ([[x=ex:a], [x=ex:b], [x=ex:c], [x=ex:d], [x=ex:e], [x=ex:f], [x=ex:g]])"
-				+ LINE_SEP,
-				q.getTupleExpr().toString());
+		assertThat(q.getTupleExpr().toString()).isEqualToNormalizingNewlines("QueryRoot\n" +
+				"   Projection\n" +
```

Comment:
Should we not have + LINE_SEP instead of \n to be consistent?

Reply:
`isEqualToNormalizingNewlines` fixes the new line seperators for us

Reply:
But might change it back actually, since it's the only place that tests the line separation aspect of the query plan.